addons/project/models/project.py

```diff
@@ -1705,6 +1715,71 @@ def _compute_display_parent_task_button(self):
         for task in self:
             task.display_parent_task_button = task.parent_id in accessible_parent_tasks
 
+    def get_group_pattern(self):
```

Comment:
Should be private otherwise the method will be accessible by portal/public users too.